### ISSUE TITLE
[BUG] set {_custom} to compound tag "custom" of (nbt compound of event-block)

### ISSUE THREAD

**Issue by Douginton** [ShaneBeee/SkBee#209]
This should work right?
set {_custom} to compound tag "custom" of (nbt compound of event-block)
But it doesn't work.

```
The error is this:
[ERROR] #!#!

[ERROR] #!#! [Skript] Severe Error:

[ERROR] #!#!

[ERROR] #!#! Something went horribly wrong with Skript.

[ERROR] #!#! This issue is NOT your fault! You probably can't fix it yourself, either.

[ERROR] #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).

[ERROR] #!#! Following plugins are probably related to this error in some way:

[ERROR] #!#! skript-reflect (https://github.com/TPGamesNL/skript-reflect)

[ERROR] #!#! You should try disabling those plugins one by one, trying to find which one causes it.

[ERROR] #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.

[ERROR] #!#! In that case, you will be given instruction on how should you report it.

[ERROR] #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.

[ERROR] #!#! Only if the author tells you to do so, report it to Skript's issue tracker.

[ERROR] #!#!

[ERROR] #!#! Stack trace:

[ERROR] #!#! java.util.ConcurrentModificationException

[ERROR] #!#! at java.base/java.util.HashMap$KeySet.forEach(Unknown Source)

[ERROR] #!#! at com.shanebeestudios.skbee.api.NBT.NBTCustomBlock.setData(NBTCustomBlock.java:40)

[ERROR] #!#! at com.shanebeestudios.skbee.api.NBT.NBTCustomBlock.getData(NBTCustomBlock.java:28)

[ERROR] #!#! at com.shanebeestudios.skbee.elements.nbt.expressions.ExprNbtCompound.lambda$get$0(ExprNbtCompound.java:80)

[ERROR] #!#! at ch.njol.skript.registrations.Converters.convert(Converters.java:390)

[ERROR] #!#! at ch.njol.skript.registrations.Converters.convertUnsafe(Converters.java:381)

[ERROR] #!#! at ch.njol.skript.expressions.base.PropertyExpression.get(PropertyExpression.java:102)

[ERROR] #!#! at com.shanebeestudios.skbee.elements.nbt.expressions.ExprNbtCompound.get(ExprNbtCompound.java:71)

[ERROR] #!#! at com.shanebeestudios.skbee.elements.nbt.expressions.ExprNbtCompound.get(ExprNbtCompound.java:36)

[ERROR] #!#! at ch.njol.skript.expressions.base.PropertyExpression.get(PropertyExpression.java:73)

[ERROR] #!#! at ch.njol.skript.lang.util.SimpleExpression.getArray(SimpleExpression.java:99)

[ERROR] #!#! at ch.njol.skript.lang.util.SimpleExpression.getSingle(SimpleExpression.java:57)

[ERROR] #!#! at com.shanebeestudios.skbee.elements.nbt.expressions.ExprTagOfNBT.get(ExprTagOfNBT.java:84)

[ERROR] #!#! at ch.njol.skript.lang.util.SimpleExpression.check(SimpleExpression.java:158)

[ERROR] #!#! at ch.njol.skript.conditions.CondCompare.check(CondCompare.java:308)

[ERROR] #!#! at ch.njol.skript.lang.Condition.run(Condition.java:54)

[ERROR] #!#! at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)

[ERROR] #!#! at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)

[ERROR] #!#! at ch.njol.skript.lang.Trigger.execute(Trigger.java:56)

[ERROR] #!#! at com.btk5h.skriptmirror.skript.reflect.EvtByReflection$MyEventExecutor.execute(EvtByReflection.java:38)

[ERROR] #!#! at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)

[ERROR] #!#! at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589)

[ERROR] #!#! at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576)

[ERROR] #!#! at org.bukkit.craftbukkit.v1_17_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:504)

[ERROR] #!#! at net.minecraft.server.level.PlayerInteractManager.a(PlayerInteractManager.java:485)

[ERROR] #!#! at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1527)

[ERROR] #!#! at net.minecraft.network.protocol.game.PacketPlayInUseItem.a(PacketPlayInUseItem.java:33)

[ERROR] #!#! at net.minecraft.network.protocol.game.PacketPlayInUseItem.a(PacketPlayInUseItem.java:1)

[ERROR] #!#! at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:30)

[ERROR] #!#! at net.minecraft.server.TickTask.run(SourceFile:18)

[ERROR] #!#! at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(SourceFile:151)

[ERROR] #!#! at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)

[ERROR] #!#! at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(SourceFile:125)

[ERROR] #!#! at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1148)

[ERROR] #!#! at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1141)

[ERROR] #!#! at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(SourceFile:134)

[ERROR] #!#! at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1125)

[ERROR] #!#! at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1054)

[ERROR] #!#! at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:305)

[ERROR] #!#! at java.base/java.lang.Thread.run(Unknown Source)

[ERROR] #!#!

[ERROR] #!#! Version Information:

[ERROR] #!#! Skript: 2.6.2 (latest)

[ERROR] #!#! Flavor: skriptlang-github

[ERROR] #!#! Date: 19:12:53.596652600

[ERROR] #!#! Bukkit: 1.17.1-R0.1-SNAPSHOT

[ERROR] #!#! Minecraft: 1.17.1

[ERROR] #!#! Java: 16.0.1 (OpenJDK 64-Bit Server VM 16.0.1+9)

[ERROR] #!#! OS: Linux amd64 4.15.0-159-generic

[ERROR] #!#!

[ERROR] #!#! Server platform: Spigot

[ERROR] #!#!

[ERROR] #!#! Current node: null

[ERROR] #!#! Current item: [[nbttype:byte tag]] "custom;wheat" of nbt compound from the block (above {_block} (as org.bukkit.Location) >> ch.njol.skript.expressions.ExprBlock$1@20cec2ef: location -> block) is equal to [[long:1]] (comparator: null)

[ERROR] #!#! Current trigger: PlayerInteractEvent (all PlayerInteractEvent with priority HIGH) (BoerFarm.sk, line 231)

[ERROR] #!#!

[ERROR] #!#! Thread: Server thread

[ERROR] #!#!

[ERROR] #!#! Language: english

[ERROR] #!#! Link parse mode: DISABLED

[ERROR] #!#!

[ERROR] #!#! End of Error.

[ERROR] #!#!
```

**Comment by ShaneBeee** [comment]
where is the rest of the template you deleted?
We are missing info here, which is why I made that template. 
I didn't make it so you could delete it and not provide info

**Comment by ShaneBeee** [comment]
im closing this due to lack of response.

That said im pretty sure this is fixed in SkBee 1.18.0-Beta2... someone else reported something like this on discord
